nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -855,15 +866,28 @@ export function RanEdit(props: Props) {
                 />
               }>
               {enbsCtx?.state &&
-                Object.keys(enbsCtx.state.enbInfo).map(enbSerial => (
-                  <MenuItem key={enbSerial} value={enbSerial}>
-                    <Checkbox checked={connectedEnodebs.includes(enbSerial)} />
-                    <ListItemText
-                      primary={enbsCtx.state.enbInfo[enbSerial].enb.name}
-                      secondary={enbSerial}
-                    />
-                  </MenuItem>
-                ))}
+                Object.keys(enbsCtx.state.enbInfo).map(enbSerial => {
+                  if (
+                    isEnodebUnregistered(
+                      enbsCtx.state.enbInfo[enbSerial].enb,
+                      props.gateway,
+                    )
+                  ) {
```

Comment:
Is this getting called on every render? It's a bit expensive to compute. See [useMemo](https://reactjs.org/docs/hooks-reference.html#usememo) hook

Reply:
I have updated it to use useCallback hook. Sorry, for the late reply. It's because I was learning the useMemo and useCallback hook and trying different alternatives like saving the enodeb state and filtering that which would have made it faster, but the saved enodeb state would not be visible when we change the access gateways. Any